### PR TITLE
Uwsgi deployment

### PR DESCRIPTION
**Pull request by nateMJ**
Addresses `Update superset role to use uWSGI application server` task in https://github.com/onaio/playbooks/issues/1921

### CODE REVIEW

File: tasks/config.yml [hunk 1]
```diff
@@ -44,6 +44,14 @@
   become_user: "{{ superset_user }}"
   command: "{{ superset_virtualenv_path }}/bin/superset init"
 
+- name: Copy uwsgi.ini
+  template:
+    src: uwsgi.ini.j2
+    dest: "{{ superset_virtualenv_path }}/uwsgi.ini"
```
Comment (morrismukiri) [inline tasks/config.yml]: Can we copy the ini file to `superset_codebase_path`?

File: defaults/main.yml [hunk 2]
```diff
@@ -201,3 +201,22 @@ superset_dashboard_position_data_limit: 65535
 superset_virtualenv_path: "{{ superset_home }}/.virtualenvs/{{ superset_user }}"
 superset_python_path: "{{ superset_virtualenv_path }}/lib/{{ superset_python_executable }}"
 superset_base_path: "{{ superset_python_path }}/site-packages/superset"
+
+# uwsgi configs
+superset_wsgi_socket: "{{ superset_codebase_path }}/superset.sock"
+superset_wsgi_chmod_socket: "664"
+superset_codebase_path: "/tmp"
```
Comment (morrismukiri) [inline defaults/main.yml]: Would it make sense if the default codebase path was `"{{ superset_home }}/app"`?

Comment (nateMJ) [reply defaults/main.yml]: Yeah, okay

File: tasks/config.yml (hunk 1 above)
Comment (nateMJ) [reply tasks/config.yml]: Would it make sense to have it under site-packages? This is superset base path https://github.com/onaio/ansible-superset/blob/master/defaults/main.yml#L203

File: defaults/main.yml (hunk 2 above)
Comment (nateMJ) [reply defaults/main.yml]: Cool

**Pull request merged**